nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -80,28 +102,31 @@ impl CompileContext<'_> {
 /// A `Send` and `Sync` wrapper over a `&'static T` where `T` might not itself
 /// be `Sync`. This is safe because the reference can only be obtained from the
 /// same thread in which the `SendableRef` was created.
-pub(crate) struct SendableRef<T: 'static> {
+pub(crate) struct SendableRef<T: ?Sized + 'static> {
     reference: &'static T,
     thread_id: std::thread::ThreadId,
 }
 
-impl<T> SendableRef<T> {
+impl<T: ?Sized> SendableRef<T> {
     pub(crate) fn new(reference: &'static T) -> Self {
         Self {
             reference,
             thread_id: std::thread::current().id(),
         }
     }
-    pub(crate) fn get(&self) -> &T {
+    pub(crate) unsafe fn new_as_static(reference: &T) -> Self {
```

Comment:
**suggestion:** A comment about the safety conditions for calling this function would not go amiss.

Reply:
Done.